action.yml

```diff
@@ -40,6 +40,8 @@ inputs:
     description: 'App: Optional Galax Pay partner hash'
   datafrete-operator-token:
     description: 'App: Random token (https://randomkeygen.com/) to validate Datafrete webhooks'
+  delay-cart-ms:
+    description: 'Events: Delay to send late cart event'
```

Comment:
```suggestion
```

A priori não botofé de colocar isso aqui não, fica como uma env mas se for o caso de editar (provavelmente não vai) tem que ser hardcoded no entry file da function mesmo...